apps/mi/dev/base/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
+namespace: mi
+patches:
+  #overlays for mi-azure-functions
+  - path: ../../mi-azure-functions/dev.yaml
```

Comment:
This doesn't look right are you missing the target on purpose?

Reply:
Took the cue from Mohana's work in sbox - maybe there's something he added in a later PR that I'm missing? https://github.com/hmcts/sds-flux-config/pull/1394/files